docs/content/preview/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md

```diff
@@ -196,10 +203,49 @@ The following is the runtime configuration to specify:
 
   Replace `[SERVICE ACCOUNT PASSWORD]` with the actual value.
 
-## Define the YugabyteDB Anywhere role
+## Role management
+
+You can map LDAP groups to [YugabyteDB Anywhere roles](../../security/authorization-platform/#roles). Upon login, the user's role is assigned based on whatever role their LDAP group has been mapped to in YugabyteDB Anywhere. If a user is in multiple mapped LDAP groups, YugabyteDB Anywhere performs a union of all the roles that are chosen as a result of group mapping and assigns the union to the user.
+
+To map LDAP groups to YugabyteDB Anywhere roles, do the following:
+
+1. Navigate to **Admin > User Management > User Authentication** and select **LDAP Configuration**.
+
+1. Under **Role Settings**, enable the **Define Role to Group Mapping** option.
+
+1. Choose how to look up LDAP group membership:
+
+    - Select **User Attribute** and set the name of the LDAP user attribute to use to find the groups that users belong to.
+    - Select **Group Search Filter** and enter an LDAP search filter to search for membership in group member listings. To specify a YugabyteDB Anywhere user in the filter, use the string `{username}`. For all occurrences of this string in the query filter, YBA will replace those with the actual username used to log in to YugabyteDB Anywhere. Enter a group search base DN to use for the group search. Use the scope option to set the scope of the search; there are three levels - SUBTREE, ONELEVEL, and OBJECT.
+
+1. Click **Create Mappings** (or, if you have existing mappings, **Edit**) to display the **Create Mapping** dialog.
+
+1. Click **Add rows** to add mappings. Select the YugabyteDB Anywhere role and enter the LDAP Group DN that you want to map the role to.
 
-You need to define a YugabyteDB Anywhere-specific role for each user on your LDAP server by setting the `yugabytePlatformRole` annotation. The value set for this annotation is read during the YugabyteDB Anywhere login. Note that if the value is modified on the LDAP server, the change is propagated to YugabyteDB Anywhere and automatically updated during login. Password updates are also automatically handled.
+1. Click **Confirm**.
+
+1. Click **Save** when you are done.
+
+### Define the YugabyteDB Anywhere role
+
+You need to define a YugabyteDB Anywhere-specific role for each user on your LDAP server by setting the `yugabytePlatformRole` annotation on the LDAP server. The value set for this annotation is read during the YugabyteDB Anywhere login. Note that if the value is modified on the LDAP server, the change is propagated to YugabyteDB Anywhere and automatically updated during login. Password updates are also automatically handled.
```

Comment:
Could be "You can also define", as this is no longer the only way to do role management via LDAP in YBA.